setup.py

```diff
@@ -42,7 +42,7 @@ def find_scripts(script_dir, prefix):
     version=version,
     packages=find_packages(),
     install_requires=[
-        "astropy>=3,<5",
+        "astropy>3,<5",
```

Comment:
using `~=4.0` would be more idiomatic

Reply:
It shows explicitly which version is wanted instead of just excluding which are not

Reply:
and what when we get 4.1?
wouldn't `astropy>3,<5` install the latest one?

Reply:
The tilde notation does the same, it allows any version `4.x` greater than the one given.

So `~=4.0` is equivalent to `>=4.0,<5`

Reply:
https://www.python.org/dev/peps/pep-0440/

Reply:
ok, thanks, I updated it!